### PR TITLE
fix: adds test for cartitems/AngelicaC

### PR DESCRIPTION
**Pull request by angie-3**
working on test. Awaiting feedback.

### CODE REVIEW

**Comment by Josephcabs** [comment]
Good work Angelica, once the modifications on the cartItems.java folder are done we can modify the tests accordingly.

File: app/src/test/CartItemTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+package q4.team1;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class AppTest {
```
Comment (XavierCruz5106) [inline app/src/test/CartItemTest.java:6]: please remove this AppTest class!

**Review by XavierCruz5106 — CHANGES_REQUESTED** [review]
Looks good just remove that one class then we can merge it!

File: app/src/test/CartItemTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        assertEquals(1.99, cartItem.getOrgPrice(), "getOrgPrice() should return the original price of 1.99");
+    }
+
+    @Test
+    public void testToString() {
+        String expected = "name: Apple price: $1.99 quantity: 5";
+        assertEquals(expected, cartItem.toString(), "toString() should return the correct format");
+    }
+
+
```
Comment (XavierCruz5106) [inline app/src/test/CartItemTest.java:68]: consider adding more tests for the price scaling with the quantity as well!

Comment (angie-3) [reply app/src/test/CartItemTest.java]: ok, I will add more test. thanks

File: app/src/test/CartItemTest.java (hunk 1 above)
Comment (angie-3) [reply app/src/test/CartItemTest.java]: completed

**Review by Josephcabs — APPROVED** [review]
I like the tests, integrates all methods. Liked the convo we had about making the methods do more in the integration tests.

**Review by XavierCruz5106 — APPROVED** [review]

**Pull request merged**